testing/python/metafunc.py

```diff
@@ -300,14 +300,35 @@ def test_idmaker_idfn_exception(self):
         def ids(val):
             raise Exception("bad code")
```

Comment:
Probably better to raise a more unique exception here, like `ValueError("error on ids")` and catch that in `pytest.raises` below, also checking that is the same message.

Otherwise in the future someone might refactor this and this raises some other exception and this tests won't get it.